docs/install/scala_setup.md

```diff
@@ -0,0 +1,87 @@
+# Setup the MXNet Package for Scala
+
+The following instructions are provided for macOS and Ubuntu. Windows is not yet available.
```

Comment:
The following instructions are provided to build from source for macOS and Ubuntu.

Reply:
Not really.

Reply:
The second half of the page goes into building from source. The actual instructions for building from source are held on the different os pages:
* http://mxnet.incubator.apache.org/install/osx_setup.html#build-the-shared-library
* http://mxnet.incubator.apache.org/install/ubuntu_setup.html#installing-mxnet-on-ubuntu
* http://mxnet.incubator.apache.org/install/windows_setup.html#build-the-shared-library